src/platform/utilities/environment/index.js

```diff
@@ -56,7 +56,7 @@ export default Object.freeze({
    * you should instead consider using the API helper function defined in
    * platform/utilities/api to fetch data.
    */
-  API_URL: environment.API_URL,
+  API_URL: 'https://staging-api.va.gov',
```

Comment:
I don't think you meant to commit this 😂

Reply:
ruh roh